packages/app/src/utils/API/nominationsAPI.js

```diff
@@ -29,6 +29,9 @@ const nominationsAPI = {
   syncNominations: function () {
     return axios.get(`${cleanURL(API_URL)}/api/syncnominations`);
   },
+  validateEmail: function (token) {
+    return axios.get(`${cleanURL(API_URL)}/api/confirmation/${token}`);
+  }
```

Comment:
A larger question: what if we had a route on the React side (`/email-verification/:token`, say) that called this endpoint instead of having the link go to the API? Seems a little cleaner, especially if the front end gets an actual domain name in production down the road.

Reply:
That good option, I was so focused on the pug file and got blinded.  I'll start working on a React route